docs/usage.md

```diff
@@ -431,16 +433,42 @@ Specify the path to a specific config file (this is a core NextFlow command).
 
 **NB:** Single hyphen (core Nextflow option)
 
-Note - you can use this to override defaults. For example, you can specify a config file using `-c` that contains the following:
+Note - you can use this to override pipeline defaults.
+
+### `--custom_config_version`
+
+Provide git commit id for custom Institutional configs hosted at `nf-core/configs`. This was implemented for reproducibility purposes. Default: `master`.
+
+```bash
+## Download and use config file with following git commid id
+--custom_config_version d52db660777c4bf36546ddb188ec530c3ada1b96
+```
+
+### `--custom_config_base`
+
+If you're running offline, nextflow will not be able to fetch the institutional config files
```

Comment:
I'm not sure about that, we should ask on slack if it's the right procedure for that

Reply:
This came from nf-core tools TEMPLATE sync I think.

Reply:
ok, then we're good